requirements.txt

```diff
@@ -17,6 +17,7 @@ pypdf
 fpdf
 youtube-transcript-api
 pytube
+openai
```

Comment:
We don't use OpenAI in our codebase. We use Google Gemini. Please, align with that.

Reply:
Resolved.